### ISSUE TITLE
Plugin doesn't work on Moodle 3.8

### ISSUE THREAD

**Issue by kirildevlion** [devlionco/moodle-local_sharewith#7]

**Comment by acquaalta** [comment]
Hi,
**Moodle version:** 3.8 (Build: 20191118)
**Theme:** Boost
**Plugin version:** INITIAL (2019042908)

TL;DR - It seems that the current version (code) of this plugin doesn't work on version 3.8 of Moodle.

When trying to copy or share an activity, the following "window" appears:

![ShareWith_Moodle38](https://user-images.githubusercontent.com/51836675/71003213-64de9900-20e9-11ea-92b7-8fcc500258e0.png)

There is no data that can be selected, so the user can't continue the process.

We verified that all the settings in the site settings area of the "Share With" plugin are enabled:

![image](https://user-images.githubusercontent.com/51836675/71004040-2bf2f400-20ea-11ea-9419-1da50e4c442f.png)

Could you please fix it?

Thanks in advance.

**Comment by ilyaDymshits** [comment]
Have done some improvements to resolve this issue
